packages/form-plugin/src/utils.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Boolean conversion:
+ *
+ *    like '', 'false' to boolean
+ *    ngxsFormClearOnDestroy, ngxsFormClearOnDestroy="true" should be treated as true
+ *    ngxsFormClearOnDestroy='false' should be treated as false
+ *
+ * @ignore
+ */
+export function coerceBoolean(val: boolean | string | undefined | null) {
+  return val != null && `${val}` !== 'false';
```

Comment:
maybe try check undefined also? and use strict (`val !== null`)

Reply:
`!= null` covers both null and undefined 😃